### DISCUSSION TITLE
Import --tw variables only

### DISCUSSION THREAD

**Discussion by jqmg** [tailwindlabs/tailwindcss#10957] [Help]
I tried looking through docs / issues / discussions best i could and could not find info on this.

Is there a way to Import --tw variables only without the preflight styles?

My use case is I am already building on top of other CSS and don't want to include the unnecessary preflight css. I just need the  --tw variables.

Please help.

**Comment by wongjn** [comment]
As long as `@tailwind base` is included somewhere in the input CSS and `experimental.optimizeUniversalDefaults` is `false` (which is the default), then the `--tw-*` variables should be compiled into the output CSS.

**Reply by jqmg** [reply]
Thank you for the prompt response. I have experimental.optimizeUniversalDefaults set to false and @tailwind base is giving me preflight styles.

**Reply by wongjn** [reply]
With the preflight core plugin enabled, its CSS rules are indeed injected into `@tailwind base`.

**Reply by jqmg** [reply]
Ah ok, I somehow missed that you can disable preflight. That is what I needed. Thank you! https://tailwindcss.com/docs/preflight#disabling-preflight
